svc/FetchService.js

```diff
@@ -78,9 +78,17 @@ export class FetchService {
             }
         }
 
-        const ret = await fetch(url, opts);
-        if (!ret.ok) throw Exception.requestError(opts, ret);
-        return ret;
+        try {
+            const ret = await fetch(url, opts);
+            if (ret && !ret.ok) throw Exception.requestError(opts, ret);
+            return ret;
+        } catch (e) {
+            if (e == 'TypeError: Failed to fetch') {
```

Comment:
The value of 'e' here seems special to me, and may vary among browsers.  This is the value in Chrome.  'e' was undefined in the console.   The value of e could only be discovered by doing console.log(e).